### PR TITLE
fix: Minor fixes on Wallet component, support custom avatar on modules components

### PR DESCRIPTION
**Pull request by cgero-eth**
## Description

- Update `ICompositeAddress` interface and components using it to support custom avatar
- Fix ENS name truncation on `Wallet` component
- Fix `Wallet` component to only resolve user ENS name when `name` property is not set
- Update `Wallet` component to support custom `chainId` and `wagmi` configurations

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

-   [x] I have selected the correct base branch.
-   [x] I have performed a self-review of my own code.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I have made corresponding changes to the documentation.
-   [x] My changes generate no new warnings.
-   [x] Any dependent changes have been merged and published in downstream modules.
-   [x] I ran all tests with success and extended them if necessary.
-   [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the [UPCOMING] title and before
        the latest version.
-   [x] I have tested my code on the test network.

### CODE REVIEW

File: src/modules/components/asset/assetTransfer/assetTransfer.tsx [hunk 1]
```diff
@@ -6,11 +6,11 @@ import { AssetTransferAddress } from './assetTransferAddress';
 
 export interface IAssetTransferProps extends IWeb3ComponentProps {
     /**
-     * Address (& optional ENS Name) of the transaction sender.
+     * Sender of the transation.
```
Comment (thekidnamedkd) [inline src/modules/components/asset/assetTransfer/assetTransfer.tsx:9]:
```suggestion
     * Sender of the transaction.
```

File: src/modules/components/wallet/wallet.stories.tsx [hunk 2]
```diff
@@ -25,24 +25,55 @@ type Story = StoryObj<typeof Wallet>;
...
         },
     },
 };
 
 /**
- * Example usage of a connected Wallet component with additional ENS handle provided.
+ * Example usage of the Wallet component with a long name.
  */
-export const WithEnsName: Story = {
+export const LongName: Story = {
```
Comment (thekidnamedkd) [inline src/modules/components/wallet/wallet.stories.tsx:72]: 😆

**Review by thekidnamedkd — DISMISSED** [review]
LGTM! Just the teeensy tiny edit. 🥇

File: src/modules/components/asset/assetTransfer/assetTransfer.tsx (hunk 1 above)
Comment (cgero-eth) [reply src/modules/components/asset/assetTransfer/assetTransfer.tsx:9]: These typos 🙈

**Pull request merged**